doc/api/fs.md

```diff
@@ -2166,6 +2166,9 @@ changes:
 * `options` {Object}
   * `bigint` {boolean} Whether the numeric values in the returned
     [`fs.Stats`][] object should be `bigint`. **Default:** `false`.
+  * `noThrow` {boolean} Whether an exception should be thrown
```

Comment:
Using a negation is not ideal for options. It's more difficult than just e.g., `throws` and using `true` as default.

Reply:
My concern was that people already passing `{ bigInt: true }` would need to update their code.

Reply:
Happy to choose a new name, I just think `undefined` has to retain the current behavior.

Reply:
How about `returnUndefinedOnError`?

Reply:
> My concern was that people already passing { bigint: true } would need to update their code.

I think @BridgeAR meant to have check like `if (options.throws === false)` so that if the user hasn't specified `false` explicitly then it would work as it is now so there won't be any breaking. (this retains the current behavior with `undefined`)

IMO `throws` is clearer for option name.

Reply:
@lundibundi If that's idiomatic, I'm happy to change it, but I have to admit that I find it confusing that the falsey values `null` and `undefined` would mean `true`.

Reply:
I understand your concern but it's not uncommon to have such cases. The API clearly states that `boolean` is expected so if non-boolean is passed (when nothing is passed `undefined` is just interpreted as default === true) it is free to just throw a `TypeError` (for i.e `null`).

Would the `ignoreErrors` option name with `false` default sound better perhaps?

Reply:
I always try to adopt local conventions.  I people are accustomed to `undefined` meaning "use the default", then `throw` is a great name.

Reply:
Done.  Thanks for clarifying!